### PR TITLE
Utility panel: Remove hard coded padding

### PR DESCRIPTION
**Pull request by raymondboswel**
## Description

The current design does not provide flexibility to consumers who may want to use the full width of the panel. Removing the padding solves this.

References #2699

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update

## How Has This Been Tested?

Storybook instance was started up, and styling inspected.

## Checklist

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules
- [x] I have checked my code and corrected any misspellings

### CODE REVIEW

File: stencil-workspace/src/components/modus-utility-panel/modus-utility-panel.scss [hunk 1]
```diff
@@ -34,13 +34,11 @@
       align-items: center;
       display: flex;
       height: 50px;
-      padding: 0 1rem;
```
Comment (prashanth-offcl) [inline stencil-workspace/src/components/modus-utility-panel/modus-utility-panel.scss:37]: Can we keep the default behavior and provide an option to override the value using css variables?

Comment (raymondboswel) [reply stencil-workspace/src/components/modus-utility-panel/modus-utility-panel.scss:37]: Doesn't that just complicate the API for little actual benefit? For a consumer to add padding to their own content is simple and natural. Having to look up a prop or set a css variable is considerably more effort. One of the main critiques for web components is difficulty in styling, thus being less opinionated is a step in the right direction IMO.

Comment (prashanth-offcl) [reply stencil-workspace/src/components/modus-utility-panel/modus-utility-panel.scss:37]: @raymondboswel I made the suggestion with the following reasons in my mind sorry for not adding them. But I agree you are right and using CSS variables for style extensibility comes with a lot of drawbacks. 

- The default behavior with padding will benefit most of the integrators, only for edge cases we will have the need to remove the default padding. 
- If the value changes in future then only the web component needs to be updated.
- Removing padding is a breaking change for other integrators.

Comment (prashanth-offcl) [reply stencil-workspace/src/components/modus-utility-panel/modus-utility-panel.scss:37]: @raymondboswel Based on the above comment I believe it is better to go with an optional property. could you please make that change?

**Pull request closed without merging**